bench/irmin-pack/trace_replay_intf.ml

```diff
@@ -61,7 +64,19 @@ module Config = struct
       blobs, instead of their actual value read in the trace.
 
       [inode_config] is a pair of ints that will be stored in the results of the
-      replay. *)
+      replay.
+
+      [gc_every] defines the distance between calls to gc. [0] disables the GC.
+
+      [gc_distance_in_the_past] defines the distance between the GC-commit and
```

Comment:
I didn't understand this doc comment for gc_distance_in_the_past initially, but after thinking about it a bit, I think I do. Maybe add something like the following for clarity? (just a suggestion)

GC is trigggered every [gc_every] commits. When GC is triggered, we select a previous commit that is [gc_distance_in_the_past] commits from the current head commit.